src/media/modules/class-wpml-beaver-builder-media-node-content-slider.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+class WPML_Beaver_Builder_Media_Node_Content_Slider extends WPML_Beaver_Builder_Media_Node {
+
+	private $property_prefixes = array(
+		'bg_',
```

Comment:
@strategio Maybe add a comment after each of these so it's clearer to see what properties you are handling.